ruby/lib/minitest/queue/runner.rb

```diff
@@ -582,6 +582,14 @@ def retry?
         ENV["BUILDKITE_RETRY_COUNT"].to_i > 0 ||
           ENV["SEMAPHORE_PIPELINE_RERUN"] == "true"
       end
+
+      def reproduce_command
+        if ENV['CI'] != 'true' || queue.is_a?(CI::Queue::File)
```

Comment:
I don't think you need this conditional given that it's expected you'd run this repro command locally.

Reply:
I think the idea was to just not raise in this case here

https://github.com/Shopify/ci-queue/blob/10d895f08eebbe562dcbb75dfbe270a0c7bc565a/ruby/lib/minitest/queue/runner.rb#L255-L258

But just always printing `bundle exec minitest-queue --queue - run --build local --worker local` may work as well.